linuxdeploy_helper.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+if [ -z "$QT_DIR" ]; then
+    QT_DIR="/usr/lib/x86_64-linux-gnu/qt5"
+fi
+
+if [ -z "$QTXML_DIR" ]; then
+    QTXML_DIR=$QT_DIR/../
+fi
+
 ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $ROOT_DIR/utils.sh
```

Comment:
env. variables sounds like the best solution. You could use `platform=$(get_platform)` from utils.sh if you want to specify default paths for other architectures (linux|linux32|linux64).